content/zh-cn/docs/reference/kubernetes-api/service-resources/ingress-class-v1.md

```diff
@@ -0,0 +1,683 @@
+  -->
+  标准的列表元数据。
+  更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+- **spec** (<a href="{{< ref "../service-resources/ingress-class-v1#IngressClassSpec" >}}">IngressClassSpec</a>)
+
+  <!--
+  Spec is the desired state of the IngressClass. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+  -->
+  Spec 是 IngressClass 的期望状态。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+
+## IngressClassSpec {#IngressClassSpec}
+
+<!--
+IngressClassSpec provides information about the class of an Ingress.
+-->
+IngressClassSpec 提供有关 Ingress 类的信息。 
+
+<hr>
+
+- **controller** (string)
+
+  <!--
+  Controller refers to the name of the controller that should handle this class. This allows for different "flavors" that are controlled by the same controller. For example, you may have different Parameters for the same implementing controller. This should be specified as a domain-prefixed path no more than 250 characters in length, e.g. "acme.io/ingress-controller". This field is immutable.
+  -->
+  Controller 是指应该处理此类的控制器名称。这允许由同一控制器控制不同”口味“。例如，对于同一个实现的控制器你可能有不同的参数。
+  这应该指定为长度不超过250个字符的域前缀路径，例如 ”acme.io/ingress-controller“。该字段是不可变的。
+
+- **parameters** (IngressClassParametersReference)
+
+  <!--
+  Parameters is a link to a custom resource containing additional configuration for the controller. This is optional if the controller does not require extra parameters.
+  -->
+  Parameters 是指向控制器中包含额外配置的自定义资源的链接。如果控制器不需要额外的属性，这是可选的。
+
+  <a name="IngressClassParametersReference"></a>
+  <!--
+  *IngressClassParametersReference identifies an API object. This can be used to specify a cluster or namespace-scoped resource.*
+  -->
+  *IngressClassParametersReference 标识一个 API 对象。这可以用来指定一个集群或者命名空间范围的资源*
+
+  <!--
+  - **parameters.kind** (string), required
+
+    Kind is the type of resource being referenced.
+  -->
+  - **parameters.kind** (string)，必需
+
+      Kind 是被引用资源的类型。
+
+  <!--
+  - **parameters.name** (string), required
+
+    Name is the name of resource being referenced.
+  -->
+  - **parameters.name** (string)，必需
+
+      Name 是被引用资源的名称。
+
+  - **parameters.apiGroup** (string)
+
+    <!--
+    APIGroup is the group for the resource being referenced. If APIGroup is not specified, the specified Kind must be in the core API group. For any other third-party types, APIGroup is required.
+    -->
+
+      APIGroup 是被引用资源的组。
+      如果未指定 APIGroup ，则被指定的 Kind 必须在核心 API 组中。
+      对于任何其他第三方类型， APIGroup 是必需的。
+
+  - **parameters.namespace** (string)
+
+    <!--
+    Namespace is the namespace of the resource being referenced. This field is required when scope is set to "Namespace" and must be unset when scope is set to "Cluster".
+    -->
+
+    Namespace 是被引用资源的命名空间。
+    当范围被设置为 ”Namespace“ 时，此字段是必需的，当范围被设置为 ”Cluster“，此字段必须取消设置。
```

Comment:
Thank you! sftim. I use these characters here because of following localization_zh‘ guide(https://kubernetes.io/zh-cn/docs/contribute/localization_zh/).  You can consult @tengqm for a better explanation.